.dependabot/config.yml

```diff
@@ -0,0 +1,13 @@
+version: 1
+update_configs:
+- package_manager: "java:maven"
+  directory: "/"
+  update_schedule: "weekly"
+  ignored_updates:
+    - match:
+        dependency_name: "org.jenkins-ci.main:jenkins-core"
+    - match:
+        dependency_name: "org.jenkins-ci.plugins*"
```

Comment:
wouldn't this block POM? updates

Reply:
Right, it will. Need a smart regexp

Reply:
Not trivial to implement unfortunately, I will think how to do it later

Reply:
No worries let's get going 😓 I think let's get vault dep moved out before it starts suggesting to update vault-java-driver